modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -134,6 +134,17 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 // CCFR here.
 
+=== Autocomplete would sometimes cause corrupt data when starting during text composition. Autocomplete no longer starts during composition.
+// #TINY-10317
+
+Previously in {productname}, the autocomplete feature did not properly interrupt or consider composing mode for keyboards such as `Korean`, leading to visual issues and data integrity problems.
+
+As a consequence, typing in composing mode could result in corrupted text when autocomplete triggered, causing unexpected characters and potential data corruption.
+
+{productname} 7.0 addresses this issue by preventing autocomplete from starting while in composing mode, ensuring that the initial moving into a new element no longer causes problems.
+
+As a result, with the new behavior, the user must first exit composing mode before autocomplete can start, ensuring that the user's text is not corrupted.
```

Comment:
@FarzadHayat I've restructured this entry to match more closly to the draft outline provided by @spocke as i feel like it was missing some of the more specific details and was a little vague.

```suggestion
Previously in {productname}, the autocomplete feature did not properly interrupt or consider composing mode for keyboards when typing in `Korean`.

For example, when typing 'neko' with `Korean` keyboard the following states will be passed through:

* `ㅜ`
* `ㅜㄷ`
* `ㅜ다`
* `ㅜ다ㅐ`

As a consequence, the autocomplete feature triggers as the composition mode changes from `ㅜ` to `ㅜㄷ`, with `ㄷ` being in composing mode. While it copies the text into a new element, it fails to interrupt the composing mode. As a result, when typing `k`, nothing appeared to happen, although in the background, the composing mode actually changes to `다`. Then, when `o` is pressed, the content of the composing mode is added, and the new character is put into composing mode instead, resulting in the text `ㅜㄷ다ㅐ`.

{productname} 7.0 addresses this issue by preventing autocomplete from starting while in composing mode. This ensures that the initial movement into a new element no longer causes problems and prevents data corruption. With the new behavior, users must first exit composing mode before autocomplete can start, thereby safeguarding against text corruption and maintaining data integrity.
```

Reply:
good point, I agree with the suggestions.